### DISCUSSION TITLE
Show off: Fastlane "slack_bot" plugin is available, useful to send direct slack message

### DISCUSSION THREAD

**Discussion by crazymanish** [fastlane/fastlane#16828] [Show and tell]
Hello @joshdholtz @janpio 
Good morning! 

This is show and tells discussion section so want to inform you about Fastlane slack_bot plugin which I build recently:)
https://github.com/crazymanish/fastlane-plugin-slack_bot

This issue already closed: 😍
https://github.com/fastlane/fastlane/issues/16764

Can you re-open the other issues, and inform those users about this plugin:
- https://github.com/fastlane/fastlane/issues/15362
- https://github.com/fastlane/fastlane/issues/15431
- https://github.com/fastlane/fastlane/issues/14071

To me, I **cant** add comment, because its showing as locked issues :(
<img width="959" alt="Screenshot 2020-07-11 at 13 45 57" src="https://user-images.githubusercontent.com/5364500/87223410-0165d280-c37d-11ea-998f-3f8b1dba0836.png">

**Comment by crazymanish** [comment]
Hi, @joshdholtz @janpio 
do you want me open PR to enhance/improve default Fastlane slack action? I can open the PR:) 

We just need to one extra param for Slack's Bot api_token :) and that will be my first Fastlane contribution 🙏🏻

**Reply by crazymanish** [reply]
btw,
I created this separate Fastlane plugin because tomorrow people can expect us to upload files with a direct message, download files etc..
which really can't really possible...but with **[Fastlane slack_bot](https://github.com/crazymanish/fastlane-plugin-slack_bot)**
we can do anything and extend bot-API at any level :)


It will be always 💯 open-source, and people are welcome to contribute :p 
https://github.com/crazymanish/fastlane-plugin-slack_bot

**Comment by joshdholtz** [comment]
Those issues have been unlocked! And feel free to create a PR anytime 😇

**Reply by crazymanish** [reply]
Thanks! Added comments into those issues! Hopefully, people will like it:)
